http/action/src/test/java/io/knotx/databridge/http/action/HttpActionTest.java

```diff
@@ -101,9 +104,13 @@ void appendActionAliasToPayload(VertxTestContext testContext, Vertx vertx)
       throws Throwable {
     // given
     HttpAction tested = successAction(vertx, VALID_JSON_RESPONSE_BODY);
+    ClientRequest clientRequest = new ClientRequest();
```

Comment:
Maybe it would be worth to extract it to some method/constant?
It looks that those couple of lines are duplicated in multiple places.